lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -394,6 +396,72 @@ def add_level_three(post, options = {})
+        post[:payment_method_options][:card][:recipient_details][:last_name] = recipient_details[:last_name] if recipient_details[:last_name]
+        post[:payment_method_options][:card][:recipient_details][:email] = recipient_details[:email] if recipient_details[:email]
+        post[:payment_method_options][:card][:recipient_details][:phone] = recipient_details[:phone] if recipient_details[:phone]
+
+        if address = recipient_details[:address]
+          post[:payment_method_options][:card][:recipient_details][:address] = {}
+          post[:payment_method_options][:card][:recipient_details][:address][:country] = address[:country] if address[:country]
+          post[:payment_method_options][:card][:recipient_details][:address][:line1] = address[:line1] if address[:line1]
+          post[:payment_method_options][:card][:recipient_details][:address][:line2] = address[:line2] if address[:line2]
+          post[:payment_method_options][:card][:recipient_details][:address][:postal_code] = address[:postal_code] if address[:postal_code]
+          post[:payment_method_options][:card][:recipient_details][:address][:state] = address[:state] if address[:state]
+          post[:payment_method_options][:card][:recipient_details][:address][:city] = address[:city] if address[:city]
+        end
+
+        if account_details = recipient_details[:account_details]
+          post[:payment_method_options][:card][:recipient_details][:account_details] = {}
+
+          if card = account_details[:card]
+            post[:payment_method_options][:card][:recipient_details][:account_details][:card] = {}
+            post[:payment_method_options][:card][:recipient_details][:account_details][:card][:first6] = card[:first6] if card[:first6]
+            post[:payment_method_options][:card][:recipient_details][:account_details][:card][:last4] = card[:last4] if card[:last4]
+          end
+
+          if (unique_identifier = account_details[:unique_identifier]) && (unique_identifier[:identifier])
+            post[:payment_method_options][:card][:recipient_details][:account_details][:unique_identifier] = {}
+            post[:payment_method_options][:card][:recipient_details][:account_details][:unique_identifier][:identifier] = unique_identifier[:identifier]
+          end
+        end
+      end
+
+      def add_aft_sender_details(post, options)
+        return unless options[:sender_details]&.is_a?(Hash)
+
+        sender_details = options[:sender_details]
+        post[:payment_method_options] ||= {}
+        post[:payment_method_options][:card] ||= {}
+        post[:payment_method_options][:card][:sender_details] = {}
+        post[:payment_method_options][:card][:sender_details][:first_name] = sender_details[:first_name] if sender_details[:first_name]
+        post[:payment_method_options][:card][:sender_details][:last_name] = sender_details[:last_name] if sender_details[:last_name]
+
```

Comment:
Could we add the other optional `sender` fields here too just in case? I see `email`, `nationality`, `occupation`, and `birth_country` listed in the docs